### PR TITLE
Improve SCSS and CSS linting

### PR DESCRIPTION
**Pull request by polybuildr**
- Use different config files for scss and css
- Disable PropertySortOrder and set SelectorDepth to 4 for both
- Disable MergeableSelector for css only

Fix T52.

### CODE REVIEW

File: .css-lint.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+scss_files: 'src/static/styles/**/*.css'
+exclude: 'src/static/styles/vendor/**'
+
+linters:
```
Comment (nisargjhaveri) [inline .css-lint.yml:4]: Is it possible to merge the common stuff and have a single file for that?

**Comment by nisargjhaveri** [comment]
Tests fails. Please check.

File: .css-lint.yml (hunk 1 above)
Comment (polybuildr) [inline .css-lint.yml:4]: `scss-lint` does not support merging of configuration files. We could maybe write something that does this for us, but this is a much simpler way right now - even with the duplication.

Comment (nisargjhaveri) [inline .css-lint.yml:4]: Okay.

**Comment by polybuildr** [comment]
The gem's name has changed from `scss-lint` to `scss_lint` and so we were on an older version. Fixed in 9a7c15e2ad, so the tests should now pass.

**Pull request merged**